scene/main/scene_tree.h

```diff
@@ -146,7 +146,8 @@ class SceneTree : public MainLoop {
 	StretchMode stretch_mode;
 	StretchAspect stretch_aspect;
 	Size2i stretch_min;
-	real_t stretch_shrink;
+	// This property is called "shrink" whenever it is exposed to users to maintain compatability
```

Comment:
This comment can be removed, the one near the project setting declaration is enough.